### PR TITLE
3.0.0-CollectionItem

### PR DESCRIPTION
**Pull request by CloudPower97**
# Description

This Pull Request is for #591.

Nothing has changed between version __0.100.2__ and __1.0.0__ of `materialize-css`, but I took the opportunity to refactor the component into a stateless one and added a convenient `avatar` prop to `CollectionItem`.

## Type of change

- [x] This change requires a documentation update

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x]  I have not generated a new package version. (the maintainers will handle that)

### CODE REVIEW

File: src/CollectionItem.js [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
+  );
+};
 
 CollectionItem.propTypes = {
   active: PropTypes.bool,
   children: PropTypes.node,
   className: PropTypes.string,
-  href: PropTypes.string
+  href: PropTypes.string,
+  avatar: PropTypes.bool
```
Comment (alextrastero) [inline src/CollectionItem.js]: could we add some description to these props?

**Comment by alextrastero** [comment]
Fixes #591

File: src/CollectionItem.js (hunk 1 above)
Comment (CloudPower97) [reply src/CollectionItem.js]: Sure, going to do that ASAP

**Comment by CloudPower97** [comment]
@alextrastero ping :smile:

File: src/CollectionItem.js [hunk 2]
```diff
@@ -1,31 +1,47 @@
...
-    const classes = {
-      'collection-item': true,
-      active: active
-    };
+  const classes = cx(
+    'collection-item',
+    {
+      active,
+      avatar
+    },
```
[the remark this answers is not in the document]
Comment (CloudPower97) [reply src/CollectionItem.js]: Sure, here we go! :smiley:

[the remark this answers is not in the document]
Comment (CloudPower97) [reply src/CollectionItem.js]: Sure, here we go! :smiley:

[the remark this answers is not in the document]
Comment (CloudPower97) [reply src/CollectionItem.js]: Sure, here we go! :smiley:

**Comment by alextrastero** [comment]
Nice!

**Pull request merged**